.github/workflows/smithy-rs-sync.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  schedule:
+    # Run at the top of the hour, every hour
+    - cron:  '0 * * * *'
+
+name: Pull in the latest changes from smithy-rs
+
+env:
+  rust_version: 1.54.0
+  rust_toolchain_components: clippy,rustfmt
+  java_version: 11
+
+jobs:
+  repo-lint:
+    name: run sync tool
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/checkout@v2
+      with:
+        repository: 'aws-labs/smithy-rs'
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: ${{ env.rust_version }}
+        components: ${{ env.rust_toolchain_components }}
+        default: true
+    - name: build sync tool
+      run: cargo build --release
+      working-directory: smithy-rs/tools/smithy-rs-sync
+    - name: run sync tool
+      run: |
+        smithy-rs/target/release/smithy-rs-sync \
+          --smithy-rs smithy-rs \
+          --aws-sdk aws-sdk-rust
+    - name: push commits
```

Comment:
any reason to not do `cargo run --release`? TBH, not a huge reason to run in release mode, not like there's any performance benefit

Reply:
I'm fine switching it over, I think I'll just need to add `../` to the paths.